index.js

```diff
@@ -7,7 +7,7 @@
 
 'use strict';
 
-const isNumber = require('is-number');
+const isNumber = (v) => typeof v === "number" || (typeof v === "string" && Number.isFinite(+v) && v.trim() !== "");
```

Comment:
This returns `true` for `NaN`. Is that intentional?

If not, adding a `v - v === 0` check fixes it:

```suggestion
const isNumber = (v) => (typeof v === "number" && v - v === 0) || (typeof v === "string" && Number.isFinite(+v) && v.trim() !== "");
```